jmix-data/eclipselink/src/main/java/io/jmix/eclipselink/impl/JmixEclipselinkJtaTransactionManager.java

```diff
@@ -31,19 +32,18 @@ public JmixEclipselinkJtaTransactionManager(String key,
         super(key, userTransaction, transactionManager);
     }
 
+    @Override
+    public void setApplicationContext(ApplicationContext applicationContext) throws BeansException {
```

Comment:
Can't guarantee that this is 100% replacement of `prepareSynchronization` overriding

Reply:
Looks good